packages/compass-e2e-tests/helpers/insert-data.ts

```diff
@@ -0,0 +1,71 @@
+import { MongoClient } from 'mongodb';
+import type { Db, MongoServerError } from 'mongodb';
+
+const CONNECTION_URI = 'mongodb://localhost:27018';
+
+async function dropDatabase(db: Db) {
+  try {
+    await db.dropDatabase();
+  } catch (err) {
+    const codeName = (err as MongoServerError).codeName;
+    if (codeName !== 'NamespaceNotFound') {
+      throw err;
+    }
+  }
+}
+
+async function createBlankCollection(db: Db, name: string) {
+  await db.createCollection(name);
+}
+
+let client: MongoClient;
+
+before(async () => {
+  client = new MongoClient(CONNECTION_URI);
+  await client.connect();
+  console.log('Connected successfully to server for inserting data');
+});
+
+after(async () => {
+  await client.close();
+});
+
+beforeEach(async () => {
+  // Drop the databases that get created by tests just in case tests failed to
+  // clean them up.
+  await dropDatabase(client.db('my-sidebar-database'));
+  await dropDatabase(client.db('my-instance-database'));
+
+  const db = client.db('test');
+
+  // Drop the entire test db where we create lots of collections during test runs
+  await dropDatabase(db);
+
+  const promises = [];
+
+  // Create some empty collections for the import tests so each one won't have
+  // to possibly drop and create via the UI every time.
+  // (named loosely after the relevant test)
+  promises.push(createBlankCollection(db, 'json-array'));
+  promises.push(createBlankCollection(db, 'json-file'));
+  promises.push(createBlankCollection(db, 'extended-json-file'));
+  promises.push(createBlankCollection(db, 'csv-file'));
+  promises.push(createBlankCollection(db, 'bom-csv-file'));
+  promises.push(createBlankCollection(db, 'numbers'));
+
+  // lots of collections to test virtual scrolling
+  for (let i = 0; i < 26; ++i) {
```

Comment:
If tests are going to bring their own fixtures then they should probably only install what they want? Creating 26 empty collections is surprisingly slow and only some sidebar and collection screen tests care about that so that things reliably have to paginate.